examples/lighting-app/nrfconnect/README.md

```diff
@@ -162,6 +165,15 @@ This section lists the User Interface elements that you can use to control and
 monitor the state of the device. These correspond to PCB components on the
 platform image.
 
+> **Important**:
+>
+> Please note that the following elements are missing on the nRF52840 Dongle:
+> **Button 2**, **Button 3**, **Button 4**, **SEGGER J-Link USB port** and
+> **NFC port with antenna attached**. Logs from a nRF52840 Dongle can be
+> collected using **nRF USB port** instead of **SEGGER J-Link USB port**.
+> Functionalities associated with the remaining missing elements are
+> inaccessible.
```

Comment:
```suggestion
> **Note**:
>
> The following Device UI elements are missing on the nRF52840 Dongle:
> **Button 2**, **Button 3**, **Button 4**, **SEGGER J-Link USB port**, and
> **NFC port with antenna attached**. You can collect logs from th nRF52840 Dongle
> using the **nRF USB port** instead of the **SEGGER J-Link USB port**.
> Functionalities associated with the remaining missing elements are
> inaccessible.
```